### ISSUE TITLE
on load collapsed code sections

### ISSUE THREAD

**Issue by kkowalsky** [tangrams/tangram-play#665]
When opening large files of YAML, like the house styles, collapsible lines of code for the sprites or font sections would be very nice.

**Comment by louh** [comment]
to clarify, you mean code should be automatically collapsed when loaded, right?

Questions to think through - how do we "know" what is the appropriate sections to be collapsed? Is there a `collapsedSections` property stored in the `example.json` data? or is it more of a generic algorithm - "for any large scene file opened, collapse these sections." If done generically, is this helpful or annoying for users that own their own very large YAML files?

**Comment by kkowalsky** [comment]
yes, this would be a way of accommodating those who want to open house styles but not be overwhelmed by its massiveness or get to see the outline of the code before digging through the weeds.
